source/introduction.txt

```diff
@@ -13,7 +13,7 @@ Introduction to MongoDB Realm for Backend and Web Developers
    :class: singlecol
 
 As a backend developer, you can use {+service+} to rapidly develop cloud-based applications.
-Realm apps can react to changes in your `{+atlas+}
+App Services apps can react to changes in your `{+atlas+}
```

Comment:
Ahh, see, "App Services apps" works in this context - worth considering swapping the {+app+} source constant to this.